CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The Change Log starts from the release 5.0.0. For older releases, see [RELEASE_N
 
 - Added pessimistic version number for all the gems in Gemfile. Now we can safely run `bundle update` to update gems with patch level updates. [#1663](https://github.com/sharetribe/sharetribe/pull/1663)
 - Added a new environment variable `delayed_job_max_run_time` which controls the maximum time for a single Delayed Job job. [#1668](https://github.com/sharetribe/sharetribe/pull/1668)
+- Added search_settings table and API access point to MarketplaceService for storing community related search settings (at this point: search type of the main search) [#1666](https://github.com/sharetribe/sharetribe/pull/1666)
```

Comment:
Now that I think it again, should we remove this entry? I mean, this PR doesn't add anything new for a open source user. It's not end-to-end ready yet. So someone reading the changelog may be confused.

Sorry about this, I know that I was the one who originally pointed out that you should add the CHANGELOG. 😞